project.clj

```diff
@@ -218,6 +218,8 @@
             [lein-parent "0.3.7"]
             [puppetlabs/i18n ~i18n-version]]
 
+  :prep-tasks [["i18n" "make"] "compile" "javac"]
```

Comment:
I don't recall how long the i18n make takes, but we have a CI job that will automatically update the POT file, we might save ourselves some duplication of effort and compilation time if we omit this and just let CI keep doing what it does.

Reply:
It's near instantaneous, because the files it creates are untouched.

I made the change like this to not remove any existing functionality.

Reply:
Ah ok, yep I took a look at clj-i18n, that's not the actual command that generates the POT file